02.calendar/cal.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+// 引数処理
+const args = process.argv.slice(2);
+let today = new Date();
+let year = today.getFullYear();
+let month = today.getMonth(); // 0-based
+
+for (let i = 0; i < args.length; i++) {
+  if (args[i] === "-y" && args[i + 1]) {
+    const y = parseInt(args[i + 1]);
+    if (y >= 1970 && y <= 2100) year = y;
```

Comment:
以下のように書いたほうが分かりやすいと思います。

```suggestion
    if (1970 <= y  && y <= 2100) year = y;
```

それから if 文は必ずブロックを伴って書いてください。https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Statements/if...else#%E8%A7%A3%E8%AA%AC